Sources/TokamakDOM/Views/Button.swift

```diff
@@ -21,8 +21,8 @@ public typealias Button = TokamakCore.Button
 
 extension Button: ViewDeferredToRenderer where Label == Text {
   public var deferredBody: AnyView {
-    AnyView(HTML("button", listeners: ["click": { _ in action() }]) {
-      Text(buttonLabel(self))
+    AnyView(HTML("button", listeners: ["click": { _ in _ButtonProxy(self).action() }]) {
```

Comment:
Is there a reason this isn’t
```suggestion
    AnyView(HTML("button", listeners: ["click": _ButtonProxy(self).action]) {
```

Reply:
None other than laziness 😄

Reply:
Turns out another reason is `error: type of expression is ambiguous without more context`.